src/lib/Components/Bidding/Components/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { StyleSheet } from "react-native"
+
+import { ButtonProps, InvertedButton } from "../../Buttons"
+
+export class Button extends React.Component<ButtonProps> {
+  render() {
+    const { style, ...props } = this.props
+
+    return <InvertedButton style={[styles.default, style]} {...props} />
+  }
+}
+
+const styles = StyleSheet.create({
+  default: {
+    // fontSize: 14,
+    // fontWeight: 'normal',
+    // fontStyle: 'normal',
+    // letterSpacing: 1,
+    // color: '#ffffff'
```

Comment:
Right now it doesn't seem like the `< InvertedButton>` component is customizable (see the warning below). I'll send a separate PR to make it more flexible.

![screen shot 2018-04-13 at 1 18 10 pm](https://user-images.githubusercontent.com/386234/38748758-2b9fc84e-3f1d-11e8-9e69-7e30841f61a5.png)